README.md

```diff
@@ -21,25 +21,47 @@ changelog file based on the released version and the release date.
       * Initial commit
     ```
 
-1. Add the Gradle plugin and configure it in *build.gradle*
-
+1. Add the Gradle plugin
+    
+    1. Using the [plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block):
+    
+    In your *setting.gradle.kts*    
+    ```kotlin
```

Comment:
I think there should be newline between the sentence and the formatted block

Reply:
Not necessary.  Document is formatted in correct way.
https://github.com/tomtom-international/updatechangelog-gradle-plugin/blob/feature/support_plugins_dsl/README.md